lib/transactions.rb

```diff
@@ -0,0 +1,25 @@
+class Transactions
+
+  attr_reader :list
+
+  def initialize
+    @list = []
+  end
+
+  def store_transaction(amount, balance)
+    self.list.push({date: format_date, amount: amount, balance: balance})
```

Comment:
It feels like this line uses a plan data structure (a hash) to represent a domain concept.  Could it use a class instead?